metadata-service/auth-impl/src/main/java/com/datahub/authorization/PolicyEngine.java

```diff
@@ -96,6 +100,9 @@ public PolicyActors getMatchingActors(
       if (actorFilter.getGroups() != null) {
         groups.addAll(actorFilter.getGroups());
       }
+      if (actorFilter.getRoles() != null) {
```

Comment:
This seems to only be used in the dead code above, this check is covered by isRoleMatch